ORStools/i18n/orstools_de.ts

```diff
@@ -725,9 +735,77 @@ p, li { white-space: pre-wrap; }
 Duplikate entfernen oder Wegpunktoptimierung abwählen.</translation>
     </message>
     <message>
-        <location filename="../gui/ORStoolsDialog.py" line="338"/>
+        <location filename="../gui/ORStoolsDialog.py" line="412"/>
         <source>The request has been aborted!</source>
         <translation>Die Anfrage wurde abgebrochen!</translation>
     </message>
+    <message>
+        <location filename="../gui/ORStoolsDialog.py" line="294"/>
+        <source>Missing Waypoints</source>
+        <translation>Fehlende Wegpunkte</translation>
+    </message>
+    <message>
+        <location filename="../gui/ORStoolsDialog.py" line="294"/>
+        <source>
+                Did you forget to set routing waypoints?&lt;br&gt;&lt;br&gt;
+                
+                Use the &apos;Add Waypoint&apos; button to add up to 50 waypoints.
+                </source>
+        <translation>Haben Sie vergessen, Wegpunkte zu setzen?&lt;br&gt;&lt;br&gt; Nutzen Sie den &quot;Wegpunkt hinzufügen&quot;-Knopf um bis zu 50 Wegpunkte hinzuzufügen.</translation>
+    </message>
+    <message>
+        <location filename="../gui/ORStoolsDialog.py" line="311"/>
+        <source>Missing API key</source>
+        <translation>Fehlender API-Key</translation>
+    </message>
+    <message>
+        <location filename="../gui/ORStoolsDialog.py" line="311"/>
+        <source>
+                Did you forget to set an &lt;b&gt;API key&lt;/b&gt; for openrouteservice?&lt;br&gt;&lt;br&gt;
+                
+                If you don&apos;t have an API key, please visit https://openrouteservice.org/sign-up to get one. &lt;br&gt;&lt;br&gt; 
+                Then enter the API key for openrouteservice provider in Web ► ORS Tools ► Provider Settings or the 
+                settings symbol in the main ORS Tools GUI, next to the provider dropdown.</source>
+        <translation>Haben Sie einen &lt;b&gt;API-Key&lt;/b&gt; für den openrouteservice gesetzt?&lt;br&gt;&lt;br&gt;
```

Comment:
Maybe passive speech is a better choice here, this might be taken less personal in the heat of an Error?

Reply:
Most certainly, but I think this is not the only place where that could be improved.
Do you reckon it's worth fixing this here now, or do we rework everything at some point?

Reply:
Maybe tackling everything at once will give a more consistent result than doing it piece by piece.